tests/base/src/main/resources/schema/elytron-client-1_4.xsd

```diff
@@ -976,6 +980,56 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="masked-password-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                A masked password definition.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:attribute name="algorithm" type="xsd:string" default="masked-MD5-DES">
+            <xsd:annotation>
+                <xsd:documentation>
+                    An algorithm encrypting the password.
```

Comment:
I think "The algorithm used to encrypt the password." would be better here.

Reply:
fixed